Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM node:12-alpine
+FROM node:22-alpine AS build
+WORKDIR /app
 
-WORKDIR /opt/cvallance/mongo-k8s-sidecar
+COPY . .
+RUN npm install
```

Comment:
`npm ci`? They _should_ behave identically here but ci is always better unless you know you want to override the package-lock.